checked_message.go

```diff
@@ -30,6 +30,51 @@ type CheckedMessage struct {
 	used   atomic.Uint32
 	lvl    Level
 	msg    string
+	cms    []*CheckedMessage
+}
+
+// MultiCheckedMessage combines one or more CheckedMessages into one, keeping
+// only those that return true from OK().
+func MultiCheckedMessage(cms ...*CheckedMessage) *CheckedMessage {
```

Comment:
Note Well Reviewers:
- I'm hoping to get away with this "MultiCheckedMessage owns its varargs slice" tradeoff...
- ...rather than needing to copy, e.g. like "zap.Tee"